src/model/consent/consent.ts

```diff
@@ -42,45 +42,62 @@
  * of 'createdAt' field may need to be tested in the future.
  */
 
-import { NotFoundError, RevokedConsentModificationError } from '../errors'
-import Knex from 'knex'
+import { NotFoundError } from '../errors'
+import Knex from 'knex';
 
 /*
  * Interface for Consent resource type
  */
 export interface Consent {
   id: string;
-  initiatorId?: string;
-  participantId?: string;
+  // participant DFSP that requested a Consent resource be made
+  participantId: string;
+  // status of the consent
   status: string;
-  credentialId?: string;
-  credentialType?: string;
-  credentialStatus?: string;
-  credentialPayload?: string;
-  credentialChallenge?: string;
+  // NOTE: not sure what purpose credentialId serves.
+  credentialId: string;
```

Comment:
I think this was the `keyHandleId` of the credential that is used to lookup the credential on the user's device.

Reply:
We can remove it, as it would be covered in `credentialPayload.id`